src/Monolog/Handler/ElasticSearchHandler.php

```diff
@@ -46,20 +54,20 @@ class ElasticSearchHandler extends AbstractProcessingHandler
     protected $options = [];
 
     /**
-     * @param Client $client  Elastica Client object
-     * @param array  $options Handler configuration
-     * @param int    $level   The minimum logging level at which this handler will be triggered
-     * @param bool   $bubble  Whether the messages that are handled can bubble up the stack or not
+     * @param \Elasticsearch\Client $client ElasticSearch Client object
+     * @param array $options Handler configuration
+     * @param int $level The minimum logging level at which this handler will be triggered
+     * @param Boolean $bubble Whether the messages that are handled can bubble up the stack or not
      */
     public function __construct(Client $client, array $options = [], $level = Logger::DEBUG, $bubble = true)
     {
         parent::__construct($level, $bubble);
         $this->client = $client;
         $this->options = array_merge(
             [
-                'index'          => 'monolog',      // Elastic index name
-                'type'           => 'record',       // Elastic document type
-                'ignore_error'   => false,          // Suppress Elastica exceptions
+                'index'        => 'monolog',      // Elastic index name
+                'type'         => 'record',       // Elastic document type
```

Comment:
I'm not sure if there is bwc or legacy reasons for wanting the type name to be `record`, but if not I'd recommend using `_doc`.

Elasticsearch is in the process of removing types entirely.  Using `_doc` will allow the URLs to remain the same once types are removed, since `_doc` is being used as part of the URL in future versions (e.g. `POST /my_index/_doc/123` is the new URL in 7.0, with `_doc` being mandatory because it is part of the URL and not a type anymore).

So for future convenience, I'd go with `_doc` if possible.

More details here: https://www.elastic.co/guide/en/elasticsearch/reference/master/removal-of-types.html